### ISSUE TITLE
Accessibility: Keyboard trap within  (Featured Data sets) tables on the page

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#2]
WCAG 2.1.2 - Level (A) Violation - No Keyboard Trap
1. Go to: data.ottawa.ca
2. When tabbing to the tables the user becomes trapped within, and must be able to tab throughout all functionality on the page. You can shift+tab backwards but unable to proceed forward.
![featured dataset tables keyboard lock](https://cloud.githubusercontent.com/assets/5271253/8010097/6349c9b2-0b7b-11e5-97e6-00a41b051218.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar in your opinion, would it be best to resolve this by having [Tab] skip over the entire table, or is an alternate solution better?

**Comment by webstandards** [comment]
@deniszgonjanin the preferred method would be to make the table grid Accessible / keyboard-navigable. You're using 'slickgrid' which according to this chart is not accessible, replacing it with a drop-in table grid that supports accessibility like dgrid may be the best solution (at least they claim keyboard accessibility when compared to slickgrid in the comparison chart, I have not tested it).
http://dgrid.io/#features

A last resort if you can't find an accessible table grid would be to allow the keyboard to skip past the grid. Technically this wouldn't pass WCAG because there would be content on the page that is inaccessible, however at least keyboard / screen reader users would be able to use the rest of the page and not get caught in a keyboard trap unable to navigate.

**Comment by deniszgonjanin** [comment]
@webstandards interesting - we'll try a couple of different grid implementations and see what works. It would be good to find a way to pass WCAG for the data tables
